build.sh

```diff
@@ -4,7 +4,11 @@ set -eux
 
 cd $(dirname $0)
 
+# Generate new README.md and exit if it differs from the current one.
+OLD_README=`mktemp`
+cp README.md $OLD_README
 cargo readme -r wee_alloc -t "$(pwd)/README.tpl" > README.md
+diff $OLD_README README.md
```

Comment:
This will fail the first time someone runs `build.sh` after updating documentation, forcing them to build twice to actually build. I think it would make more sense to do this diffing in `test.sh`.